src/components/Pages/Submit/Form/Tags/index.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { FC, useCallback, useEffect, useRef, useState } from "react";
+import { Tag, TagGroup, Input } from "rsuite";
+import { ArticleParams } from "../store/types";
+import { AddLink } from "../../../../Icons";
+
+const removeTag = (
+  tag: string,
+  tags: string[],
+  upsert: (update: ArticleParams) => void
+) => {
+  const nextTags = tags.filter((item) => item !== tag);
+  upsert({ tags: nextTags });
+};
+
+const Tags: FC<{
```

Comment:
This is a good example when react state makes things easier. It listens to state changes for displaying the new tags using rsuit and hides the input.